gitlab/v4/objects/users.py

```diff
@@ -443,6 +443,7 @@ class UserProjectManager(ListMixin, CreateMixin, RESTManager):
             "description",
             "builds_enabled",
             "public_builds",
+            "public_jobs",
```

Comment:
Is this the API for this?  https://docs.gitlab.com/ee/api/projects.html#create-project-for-user

I'm guessing a bug in the API docs as they only list `public_builds` for that call.

Reply:
Answering my own question:

https://gitlab.com/gitlab-org/gitlab/-/blob/5f16f0762a05506be7fd11760f0c8d82ad0068d8/lib/api/entities/project.rb#L92

Looks like all `public_builds` now have an alias of `public_jobs`